packages/hoppscotch-sh-admin/src/composables/useConfigHandler.ts

```diff
@@ -255,7 +259,21 @@ export function useConfigHandler(updatedConfigs?: Config) {
     return config;
   });
 
-  // Trasforming the working configs back into the format required by the mutations
+  // Checking if any of the config fields are empty
+  const isFieldEmpty = (field: string) => field === '';
```

Comment:
Can a field be `null` or `undefined`?

Reply:
If it can be `null` or `undefined` then we can just check `falsy` value like this:

```
const isFieldEmpty = (field: string) => !field;
```

Reply:
@anwarulislam since the defined type here forbids null/undefined, I don't think we need to check for that.

Instead the trim check is enough here.